docs-source/content/faq/resource-settings.md

```diff
@@ -0,0 +1,162 @@
+
+If a CPU request and limit are _not_ configured for a WebLogic Server pod:
+- The pod can end up using all CPU resources available on its node and starve other containers from using shareable CPU cycles. 
+
+- The WebLogic server JVM may choose an unsuitable garbage collection (GC) strategy.
+
+- A WebLogic Server self-tuning work-manager may incorrectly optimize the number of threads it allocates for the default thread pool. 
+
+It's also important to keep in mind that if you set a value of CPU core count that's larger than core count of your biggest node, then the pod will never be scheduled. Let's say you have a pod that needs 4 cores but you have a kubernetes cluster that's comprised of 2 core VMs. In this case, your pod will never be scheduled.  
+
+### Operator sample heap and resource configuration
+
+The operator samples configure non-default minimum and maximum heap sizes for WebLogic server JVMs of at least 256MB and 512MB respectively. You can edit a sample's template or domain resource `resources.env` `USER_MEM_ARGS` to have different values. See [Configuring heap size](#configuring-heap-size).
+
+Similarly, the operator samples configure CPU and memory resource requests to at least `250m` and `768Mi` respectively.
+
+There's no memory or CPU limit configured by default in samples and so the default QoS for sample WebLogic server pod's is `Burstable`. 
+
+If you wish to set resource requests or limits differently on a sample domain resource or domain resource template, see [Setting resource requests and limits in a domain resource](#setting-resource-requests-and-limits-in-a-domain-resource). Or for samples that generate their domain resource using an 'inputs' file, see the `serverPodMemoryRequest`, `serverPodMemoryLimit`, `serverPodCpuRequest`, and `serverPodCpuLimit` parameters in the sample's `create-domain.sh` input file.
+
+### Configuring CPU affinity
+
+A Kubernetes hosted WebLogic server may exhibit high lock contention in comparison to an on-premise deployment. This lock contention may be due to lack of CPU cache affinity and/or scheduling latency when workloads move between different CPU cores.  
+
+In an on-premise deployment, CPU cache affinity, and therefore reduced lock contention, can be achieved by binding WLS java process to particular CPU core(s) (using the `taskset` command).
+
+In a Kubernetes deployment, similar cache affinity can be achieved by doing the following:
+- Ensuring a pod's CPU resource request and limit are set and equal (to ensure a `guaranteed` QoS).
+- Configuring the `kubelet` CPU manager policy to be `static` (the default is `none`). See [Control CPU Management Policies on the Node](https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies). 
+Note that some Kubernetes environments may not allow changing the CPU management policy.
+
+### Measuring JVM heap, pod CPU, and pod memory
+
+TBD Discuss/link to Grafana/Prometheus.
```

Comment:
This is a good idea... You can also link to OKE monitoring.

Reply:
We added links to "Monitoring a SOA domain" which provides steps for setting up Prometheus and Grafana in order to monitor pod memory and CPU resources and setting up WebLogic monitoring exporter for JVM heap monitoring.

Reply:
Also, added links to "Tools for Monitoring Resources" in the Kubernetes documentation. We didn't find an option to monitor pod level resources in OKE monitoring. There's a section about monitoring OKE cluster and section about node level monitoring in OCI IaaS documentation.  Please let me know if we need to link a particular section in OKE documentation. Thanks.